app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,6 @@
     <string name="hello_blank_fragment">This is a placeholder for an OAuth login to FxA</string>
     <string name="fxa_login_logmein_btn">Log Me In</string>
     <string name="settings_go_here">settings go here!</string>
+    <string name="placeholder_unlock_button">Unlock one\'s Firebox</string>
```

Comment:
ahem (-;

Reply:
Including the app_name is something we have to do by hand, in this `strings.xml`. )-:

Reply:
:+1:

Copied other resources.